docs/standard/basetypes/customnumeric.md

```diff
@@ -0,0 +1,645 @@
+---
```

Comment:
According to that material file naming guidelines haven't been finalized, so let's wait for that and enter a new issue if necessary.